bash_completion

```diff
@@ -2447,7 +2447,8 @@ _comp__included_ssh_config_files()
     # https://github.com/openssh/openssh-portable/blob/5ec5504f1d328d5bfa64280cd617c3efec4f78f3/readconf.c#L2240
     local max_depth=16
     while ((${#included[@]} > 0 && depth++ < max_depth)); do
-        _comp_split include_files "$(command sed -ne 's/^[[:blank:]]*[Ii][Nn][Cc][Ll][Uu][Dd][Ee][[:blank:]]\(.*\)$/\1/p' "${included[@]}")" || return
+        _comp_split include_files "$(command sed -n -e 's/[[:blank:]]*#.*//' -e 's/^[[:blank:]]*[Ii][Nn][Cc][Ll][Uu][Dd][Ee][[:blank:]]*\(..*\)/\1/p' "${included[@]}")" || return
```

Comment:
Should we remove the trailing/preceding whitespaces? Those are stripped by `_comp_split` anyway. Then, the sed expression can be simplified:

```suggestion
        _comp_split include_files "$(command sed -ne 's/^[[:blank:]]*[Ii][Nn][Cc][Ll][Uu][Dd][Ee][[:blank:]]*\([^#]*\).*$/\1/p' "${included[@]}")" || return
```

Reply:
this is mainly because I discovered OpenSSH maintainers, in their parser, are actually proceeding like this, simply stopping processing as soon as they hit a `#` sign and trimming
```
	/* Remove comment and trailing whitespace */
	if ((cp = strchr(orig, '#')) != NULL)
		*cp = '\0';
	rtrim(orig);
```
https://github.com/openbsd/src/blob/master/usr.bin/ssh/readconf.c#L3229-L3232

I can revert to your proposition, which, incidentally is very similar to my first proposal in the PR

Reply:
Does the result change with this implementation detail?

Reply:
I didn't get the response, but I applied the suggestion since that is the minimal change to the current main, and it is also simpler.